api/config/v1/flags.go

```diff
@@ -107,6 +107,7 @@ type GFDCommandLineFlags struct {
 	NoTimestamp     *bool     `json:"noTimestamp"     yaml:"noTimestamp"`
 	SleepInterval   *Duration `json:"sleepInterval"   yaml:"sleepInterval"`
 	OutputFile      *string   `json:"outputFile"      yaml:"outputFile"`
+	ImexNodesConfig *string   `json:"imexNodesConfig" yaml:"imexNodesConfig"`
```

Comment:
This is a nit: Should this be `imexNodesConfigFile` since we use `File` for other properties that are paths.